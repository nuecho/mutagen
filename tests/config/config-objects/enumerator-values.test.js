cfgObjectTests("enumeratorValues", "enumerator-values-config.json");
