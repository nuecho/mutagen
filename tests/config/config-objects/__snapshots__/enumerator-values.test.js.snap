// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen enumeratorValues config import-export] should create the enumeratorValues: the enumeratorValues have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ enumeratorvalue => name: 'cfgObjectTest', enumerator: 'Environment/MediaType'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen enumeratorValues config import-export] should throw exception when new enumeratorValues to import miss mandatory properties enumeratorValues: exception has been thrown 1`] = `
Object {
  "stderr": "Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
	Mandatory properties [displayName, tenant] not set in enumeratorvalue 'name: 'emptyCfgObjectTest', enumerator: 'null/MediaType''
",
  "stdout": "",
}
`;

exports[`[mutagen enumeratorValues config import-export] should update the modified enumeratorValues: the modified enumeratorValues have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ enumeratorvalue => name: 'cfgObjectTest', enumerator: 'Environment/MediaType'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen enumeratorValues config import-export] should update the same enumeratorValues: the enumeratorValues have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ enumeratorvalue => name: 'cfgObjectTest', enumerator: 'Environment/MediaType'
Completed. 1 object(s) imported.
",
}
`;
